interface/orders/orders_results.php

```diff
@@ -766,20 +766,20 @@ function validate(f) {
     if ($reviewauth) {
         ?>
      <center><p>
-         <input type='submit' name='form_submit' value='<?php xl('Sign Results', 'e'); ?>' />
+         <input type='submit' name='form_submit' value='<?php xlt('Sign Results', 'e'); ?>' />
      </p></center>
     <?php
     } else {
         ?>
      <center><p>
-         <input type='button' name='form_submit' value='<?php xl('Sign Results', 'e'); ?>' onclick="alert('<?php xl('Not authorized', 'e') ?>');" />
+         <input type='button' name='form_submit' value='<?php xlt('Sign Results', 'e'); ?>' onclick="alert('<?php xlt('Not authorized', 'e') ?>');" />
      </p></center>
     <?php
     }
 } else {
 ?>
  <center><p>
-  <input type='submit' name='form_submit' value='<?php xl('Save', 'e'); ?>' />
+  <input type='submit' name='form_submit' value='<?php xlt('Save', 'e'); ?>' />
```

Comment:
This is within an html attribute, so use xla. Also need to remove the `e` parameter and add the `echo`